libs/coin-modules/coin-polkadot/src/logic/broadcast.ts

```diff
@@ -1,7 +1,18 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
 import polkadotAPI from "../network";
 import { loadPolkadotCrypto } from "./polkadot-crypto";
 
-export default async function broadcast(signature: string): Promise<string> {
+export default async function broadcast(
+  signature: string,
+  useNode: boolean = false,
+): Promise<string> {
   await loadPolkadotCrypto();
-  return await polkadotAPI.submitExtrinsic(signature);
+  if (useNode) {
+    return await polkadotAPI.submitExtrinsic(signature);
+  }
+
+  const wsProvider = new WsProvider("ws://85.208.51.8:36003");
+  const api = await ApiPromise.create({ provider: wsProvider });
```

Comment:
Network and other external tools shouldn't be set inside *logic* component.
This instance should be define directly inside `broadcastToNode` method.